can-bus/json/PDM/PDM_alerts.json

```
@@ -1,6 +1,17 @@
 {
-    "msg_id": 400,
-    "alerts": [
-        "PDM_ALERT_STARTUP"
+    "warnings_id": 400,
+    "faults_id": 401,
+    "warnings": [
+        "PDM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_1HZ",
+        "PDM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_100HZ",
+        "PDM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_1KHZ",
+        "PDM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_CANRX",
+        "PDM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_CANTX",
+        "PDM_WARNING_WATCHDOG_TIMEOUT",
+        "PDM_WARNING_TX_OVERFLOW",
+        "PDM_WARNING_RX_OVERFLOW"
+    ],
+    "faults": [
+        "PDM_FAULT_DUMMY"
     ]
 }
```

nit: missing blank line
```suggestion
}

```